lib/transfer_jobs/sidekiq_mover.rb

```diff
@@ -47,11 +47,11 @@ def transfer(&filter)
       end
 
       logger.info "Finished recovering single queue. queue_key=#{source.key}" \
-        " num_moved=#{num_moved} num_dropped=#{num_dropped}"
+      " num_moved=#{num_moved} num_dropped=#{num_dropped}"
```

Comment:
this function (`sidekiq_mover.rb/transfer_jobs`) and the one in `queue_mover` (https://github.com/Shopify/transfer_jobs/blob/master/lib/transfer_jobs/queue_mover.rb#L15) are almost identical.

Reply:
Yep, they serve the exact same purpose for sidekiq / resque. I think converging them eventually is a good idea just not a priority right now either.